### PR TITLE
Update orgs.js

### PR DESCRIPTION
**Pull request by crandmck**
Remove abandoned, deprecated, or irrelevant repos per #49
@rmg @sam-github @chandadharap @raymondfeng @ritch

### CODE REVIEW

File: orgs.js [hunk 1]
```diff
@@ -357,20 +323,10 @@
...
-          "type": "repo"},
-        {"name": "strongloop/strong-mesh-demo",
-          "type": "repo"},
         {"name": "strongloop/loopback-example-mixins",
           "type": "repo"},
         {"name": "strongloop/loopback-acl-route",
           "type": "repo"},
-        {"name": "strongloop/strong-db-watcher",
-          "type": "repo"},
-        {"name": "strongloop/ephemeral-npm",
```
Comment (rmg) [inline orgs.js]: This is actively used. It's actually a very important part of our CI.

**Comment by rmg** [comment]
Personally, I don't think any of them should be removed unless the repos themselves have been deleted. I consider ibm.github.io an inventory of open source repositories, not a tool for promoting specific repositories.

**Comment by crandmck** [comment]
Well, I'm not sure what the point is of listing repos that haven't been touched in several years, or have no meaningful content, or are based on long-outdated and abandoned project ideas.  Including useless repos like that detracts from the repos that are current.   The number of repos is long enough already that the home page crashes Chrome, so there is clearly a nonzero cost to adding a repo to the list.

**Comment by crandmck** [comment]
@rmg I added those repos back.

**Comment by chandadharap** [comment]
@rmg We already have had a discussion about removing old deprecated repos form github. I don't see why we need to list those here?

**Comment by crandmck** [comment]
@christo4ferris I believe this can be merged, based on the @chandadharap comment above.  She is the engineering director for StrongLoop.

**Comment by bnoordhuis** [comment]
Does orgs.js have a concept of "old, archived, unmaintained"?  If not, perhaps now is a good time to add that.

**Comment by sam-github** [comment]
Why is there a list at all? github APIs allows introspection to find all repos in an org.

**Pull request merged**